obstacle_adder.py

```diff
@@ -0,0 +1,118 @@
+import glob
+import cv2 as cv
+import numpy as np
+import utils
+from random import getrandbits
+
+
+img = None
+new_img = None
+mask = None
+new_mask = None
+obs = None
+obs_mask = None
+
+
+
+def obstacles_from_folder(folder, img_ext='.png'):
+    glob_selector = '{}/*{}'.format(folder, img_ext)
+    for file in glob.glob(glob_selector):
+        yield cv.imread(file)
+
+
+def load_obstacles(folder, img_ext='png'):
+    obstacles = []
+    obstacles_mask = []
+    for obs in obstacles_from_folder(folder):
+        mask = cv.cvtColor(obs, cv.COLOR_BGR2GRAY)
+        _, mask = cv.threshold(mask, 10, 255, cv.THRESH_BINARY)
+        obstacles.append(obs)
+        obstacles_mask.append(mask)
+    return np.asarray(obstacles), np.asarray(obstacles_mask)
+
+
+def __insert_image(image, mask, x, y, h, w):
+    global obs_mask, obs
+    image = cv.copyMakeBorder(image, h, h, w, w,
+                              cv.BORDER_CONSTANT,
+                              value=[0, 0, 0])
+    mask = cv.copyMakeBorder(mask, h, h, w, w,
+                             cv.BORDER_CONSTANT,
+                             value=[0, 0, 0])
+    roi = image[y:y+h, x:x+w]
+    mask_roi = mask[y:y+h, x:x+w]
+    obs_mask_inv = cv.bitwise_not(obs_mask)
+    roi_bg = cv.bitwise_and(roi, roi, mask=obs_mask_inv)
+    roi_fg = cv.bitwise_and(obs, obs, mask=obs_mask)
+    mask_bg = cv.bitwise_and(mask_roi, mask_roi, mask=obs_mask_inv)
+    mask_fg = cv.bitwise_and(obs_mask, obs_mask, mask=obs_mask_inv)
+    roi = cv.add(roi_bg, roi_fg)
+    obs_mask_inv = cv.add(mask_bg, mask_fg)
+    image[y:y+h, x:x+w] = roi
+    mask[y:y+h, x:x+w] = obs_mask_inv
+    return image[h:-h, w:-w], mask[h:-h, w:-w]
+
+
+def visualize_labels(img, mask):
```

Comment:
@Foxiy This looks like something that could be part of utils.py.

I am also not a huge fan of calling `cv.imshow` from functions that might be imported from random places. If that call is really unavoidable, we should at least let the caller change the title of the frame.